### PR TITLE
feat: kompose compatibility with k8

### PR DESCRIPTION
**Pull request by rahulguptajss**

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -104,3 +104,26 @@ To upgrade Harvest, use the `restart` command - a newer image of Harvest will be
...
+
+Execute below commands to run harvest artifacts in kubernetes
+
+1. ```bin/harvest generate docker full -p --output harvest-compose.yml```
+2. ```kompose convert -f harvest-compose.yml -f prom-stack.yml -o kub.yaml --volumes hostPath```
+3. ```kubectl apply -f kub.yaml```
+
+#### Stop all containers
+
+```kubectl delete -f kub.yaml```
```
Comment (cgrinds) [inline docker/README.md:120]: kubectl delete --file kub.yaml

File: docker/README.md [hunk 2]
```diff
@@ -104,3 +104,26 @@ To upgrade Harvest, use the `restart` command - a newer image of Harvest will be
...
+
+### Helm Chart
+
+Generate helm charts with below command
+
+```
+kompose convert -f harvest-compose.yml -f prom-stack.yml -c --volumes hostPath
```
Comment (cgrinds) [inline docker/README.md:127]: `kompose convert --file harvest-compose.yml --file prom-stack.yml --chart --volumes hostPath`

**Review by cgrinds — APPROVED** [review]
looks good @rahulguptajss few minor documentation changes

File: docker/README.md [hunk 3]
```diff
@@ -104,3 +104,26 @@ To upgrade Harvest, use the `restart` command - a newer image of Harvest will be
...
+## Kubernetes Deployment
+
+### Requirements
+- Kompose: `v1.25` or higher https://github.com/kubernetes/kompose/
+
+Execute below commands to run harvest artifacts in kubernetes
+
+1. ```bin/harvest generate docker full -p --output harvest-compose.yml```
+2. ```kompose convert -f harvest-compose.yml -f prom-stack.yml -o kub.yaml --volumes hostPath```
+3. ```kubectl apply -f kub.yaml```
```
[the remark this answers is not in the document]
Comment (rahulguptajss) [reply docker/README.md]: kubectl apply --filename kub.yaml

File: docker/README.md (hunk 1 above)
Comment (rahulguptajss) [reply docker/README.md]: kubectl delete --filename kub.yaml

**Review by cgrinds — APPROVED** [review]

**Pull request merged**